### PR TITLE
Add df1-html

### PR DESCRIPTION
**Pull request by melisaldiaz**
I wrote a function for rendering Df1.Logs as HTML using Xmlbf  and a parser for Df1.Logs from Xmlbf HTML. 

I included .css files for the theme Solarized, both dark and light. 

I believe this library can be useful to people who need to either parse or render Df1.Logs.

### CODE REVIEW

File: df1-html/lib/Df1/Html/Parse.hs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+attrClass t = do
+  attrs <- X.pAttr "class"
+  case elem t (T.words attrs) of
+    False -> fail ("Expected \"class\" value to contain " <> show t <> ".")
+    True -> pure ()
+
+parseTime :: X.Parser TL.Text
+parseTime = X.pElement "span" $ do 
+  attrClass "df1-time"
+  X.pText <|> pure ""
```
Comment (k0001) [inline df1-html/lib/Df1/Html/Parse.hs:43]: This shouldn't fallback to succeeding with empty text. Contrary to things like `Message`, the timestamp can't be empty.

File: df1-html/lib/Df1/Html/Parse.hs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+parseLevel :: X.Parser TL.Text
+parseLevel = X.pElement "span" $ do 
+  attrClass "df1-level"
+  X.pText <|> pure ""
```
Comment (k0001) [inline df1-html/lib/Df1/Html/Parse.hs:48]: This shouldn't fallback to succeeding with empty text. Contrary to things like `Message`, the level can't be empty.

File: df1-html/lib/Df1/Html/Parse.hs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+parsePaths :: X.Parser TL.Text
+parsePaths = X.pElement "span" $ do 
+  attrClass "df1-path"
+  TL.intercalate " " <$> many (parsePush <|> parseAttr)
+
+parsePush :: X.Parser TL.Text
+parsePush = X.pElement "span" $ do
+  attrClass "df1-push"
+  t <- X.pText <|> pure ""
```
Comment (k0001) [inline df1-html/lib/Df1/Html/Parse.hs:58]: This shouldn't fallback to succeeding with empty text. Contrary to things like `Message`, this (essentially, `/`) can't be empty.

File: df1-html/lib/Df1/Html/Parse.hs [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+parseSeg :: X.Parser TL.Text
+parseSeg = X.pElement "span" $ do 
+  attrClass "df1-seg"
+  X.pText <|> pure ""
+
+parseAttr :: X.Parser TL.Text
+parseAttr = X.pElement "span" $ do
+  attrClass "df1-attr"
+  k <- parseKey
+  eq <- X.pText <|> pure ""
```
Comment (k0001) [inline df1-html/lib/Df1/Html/Parse.hs:71]: This shouldn't fallback to succeeding with empty text.

File: df1-html/lib/Df1/Html/Render.hs [hunk 5]
```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Df1.Html.Render
```
Comment (k0001) [inline df1-html/lib/Df1/Html/Render.hs:4]: Excellent work with the documentation on this module!!!

Comment (melisaldiaz) [reply df1-html/lib/Df1/Html/Render.hs:4]: Thank you!! I now corrected the parsers that succeded with empty text.

**Comment by k0001** [comment]
Excellent! Thank you!!!

**Pull request merged**